zeebe/exporters/elasticsearch-exporter/pom.xml

```diff
@@ -81,6 +86,11 @@
       <artifactId>jackson-annotations</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
```

Comment:
:x: Generally avoid adding dependencies to a module (even if they're technically used elsewhere) if it's only for small things (like equals/hashcode implementations). Every time this means coupling us more to this dependency, so we should evaluate if it's worth it (and in this case, probably not). The same for the other modules were this was added due to the hashcode/equals implementations (and please also change those).

Reply:
Removed